### DISCUSSION TITLE
Support for DuckDuckGo browser

### DISCUSSION THREAD

**Discussion by hrbeavers** [aclap-dev/video-downloadhelper#1561] [Help and Q&A]
### Your question

Did a rather thorough search and found no mention of the DuckDuckGo browser, which seems to use a number of Edge components. Is there a plug-in for the DuckDuckGo browser?  Currently using Edge browser specifically for the DownloadHelper and the companion helper. Would like to do the same in DuckDuckGo, and I realize additional payment would be required.
Thank you for your time.

### Addon & CoApp version and details

Version 9.1.1.0 (stable), target microsoft, locale en-US.

Companion App installed (v2.0.19) Recheck

C:\Program Files\DownloadHelper CoApp\vdhcoapp.exe

**Comment by Mattmjs** [comment]
If it's chromium based then you should be able to add the chrome extension, that would mean you could get a chrome license.
